server-openid4vci/src/main/java/com/android/identity/server/openid4vci/QrServlet.kt

```diff
@@ -0,0 +1,48 @@
+package com.android.identity.server.openid4vci
+
+import com.google.zxing.qrcode.decoder.ErrorCorrectionLevel
+import com.google.zxing.qrcode.encoder.Encoder
+import jakarta.servlet.http.HttpServletRequest
+import jakarta.servlet.http.HttpServletResponse
+import java.awt.image.BufferedImage
+import java.awt.image.DataBufferByte
+import javax.imageio.ImageIO
+
+
+/**
+ * Servlet that encodes text string (passed as `q` parameter) into a QR code.
+ */
+class QrServlet : BaseServlet() {
+    override fun doGet(req: HttpServletRequest, resp: HttpServletResponse) {
+        val str = req.getParameter("q") ?: ""
+        /*
+        val matrix = MultiFormatWriter().encode(str, BarcodeFormat.QR_CODE, 800, 800)
+        val w = matrix.width
+        val h = matrix.height
+        val image = BufferedImage(matrix.width, matrix.height, BufferedImage.TYPE_BYTE_GRAY)
+        val pixels = (image.raster.dataBuffer as DataBufferByte).data
+        var index = 0
+        for (y in 0 until h) {
+            for (x in 0 until w) {
+                pixels[index++] = if (matrix[x, y]) 0 else -1
+            }
+        }
+
+         */
+
+        val qr = Encoder.encode(str, ErrorCorrectionLevel.L, null)
+        val matrix = qr.matrix
+        val w = matrix.width
```

Comment:
nit: remove w and h? We aren't using them everywhere (some places we use matrix.width and matrix.height, other places we use w and h). Let's just be consistent and always use matrix.width, matrix.height.

Or, if we want to consistently use these two variables, rename them to "width" and "height." (generally avoid abbreviations)

Reply:
Yeah, fixed this.